samples/OriginalUriBaseIds.sarif

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "properties": {
+    "comment": "This sample demonstrates the use of originalUriBaseIds."
+  },
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "SarifSamples"
+        }
+      },
+      "originalUriBaseIds": {
+        "PROJECTROOT": {
+          "description": {
+            "text": "The root directory for all project files."
+          },
+          "properties": {
+            "comment": "The SARIF producer has chosen not to specify a URI for PROJECTROOT. See §3.14.14, NOTE 1, for an explanation."
+          }
+        },
+        "SRCROOT": {
+          "uri": "src",
+          "uriBaseId": "PROJECTROOT",

```

Comment:
>PROJECTROOT [](start = 24, length = 11)

Ah, now I understand, by project root you mean, ENLISTMENTROOT. Suggest making that change. Or perhaps use REPOROOT . Project is just too overloaded a term in MS dev tooling to provide a clear example here. To an old MS workhorse like me, that is. :P

Reply:
Renamed to REPOROOT.

---
In reply to: [414791357](https://github.com/microsoft/sarif-tutorials/pull/14#discussion_r414791357) [](ancestors = 414791357)